README.md

```diff
@@ -152,10 +152,23 @@ Tady bude ukázka jak Foreman docs používají AsciiDoc pro generace své dokum
 Tady si předáme slovo
 -->
 
-# 🎉
-<!--
-_class:
- - lead
- - invert
--->
-### Otázky a odpovědi
+## Demo ukázka
+
+Jak rychle a jednodušše začít psát knihy způsobem *as-code* pomocí [Githubu](github.com) a *Github pages*:
+
+* založení repozitáře
```

Comment:
Jestli vymenis `*` za `-`, cely slajd se ukaze najednou

Reply:
Klidně to uprav. Chtěl bych, aby se celý slajd zobrazil najednou.

Reply:
Hotovo